finance/views.py

```diff
@@ -49,6 +64,14 @@ def get_context_data(self, *args, **kwargs):
 
 
 class GetFinanceStartRedirectView(FeatureFlagMixin, RedirectView):
+    """
+    Using a redirect URL here instead of putting a hyperlink directly to Pardot
+    in the html. the reason is go Google links to this page instead of Pardot.
```

Comment:
whut

Reply:
as per our convo I turned a github comment into a code comment. will edit the comment to make ti clearer.